packages/jellyfish-api-core/__tests__/category/governance/createCfp.test.ts

```diff
@@ -0,0 +1,138 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import BigNumber from 'bignumber.js'
+import { UTXO, ProposalStatus, ProposalType } from '../../../src/category/governance'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+
+describe('Governance withthout WalletCoinbaseMaturity', () => {
+  const container = new MasterNodeRegTestContainer(undefined, 'defi/defichain:HEAD-a61f615')
+  const client = new ContainerAdapterClient(container)
+  let payoutAddress: string
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+
+    payoutAddress = await container.call('getnewaddress')
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should not createCfp while still in Initial Block Download', async () => {
+    const data = {
+      title: 'Testing new community fund proposal',
+      amount: new BigNumber(100),
+      cycles: 2,
+      payoutAddress
+    }
+    const promise = client.governance.createCfp(data)
+    await expect(promise).rejects.toThrow(RpcApiError)
+    await expect(promise).rejects.toThrow('RpcApiError: \'Cannot create a cfp while still in Initial Block Download\', code: -10, method: createcfp')
+  })
+
+  it('should not createCfp with insufficient fund', async () => {
+    const data = {
+      title: 'Testing new community fund proposal',
+      amount: new BigNumber(100),
+      cycles: 2,
+      payoutAddress
+    }
+    await container.generate(1)
```

Comment:
Usually, we add `await container.generate(1)` after executing a RPC to confirm the transaction
You should remove this line

Reply:
Without this line, the container stays in 'Initial Block Download'.
Is there a better way to handle this ?

Reply:
Try adding it to line 31, which is just after `const promise = client.governance.createCfp(data)`

Reply:
In that case, the promises don't throw

Reply:
Hmm, you are right. should not apply to promise

Reply:
Without this line, the container stays in 'Initial Block Download'.
Is there a better way to handle this ?

@fuxingloh

Reply:
I use this `await container.generate(1)` to generate a block in order get out of Initial Block Download.
What would be the correct way to do it ? @fuxingloh

Reply:
put container.generate in beforeAll